.circleci/config.yml

```diff
@@ -593,6 +593,9 @@ jobs:
       - *setup_bazel_remote_execution
       - *yarn_install
       - *setup_bazel_binary
+
+      - run: yarn integration-tests:nglinker
```

Comment:
Can we consider changing the name here to be `partial-ivy` rather than `linker`? The thing we are testing is whether we can consume the library compiled in the partial Ivy format. This then aligns with the following integration test that is testing that we can consume the library compiled in View Engine format.